app/controllers/inquiry/answers_controller.rb

```diff
@@ -42,6 +42,17 @@ def send_csv(items)
         filename: "inquiry_answers_#{Time.zone.now.to_i}.csv"
     end
 
+    def send_afile(file)
+      filedata = []
+      filepath = file.path
+      File.open(filepath, 'rb') do |of|
+        filedata = of.read()

```

Comment:
__[rubocop]__
* Do not use parentheses for method calls with no arguments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)

<!-- Comment by [SideCI](https://sideci.com) -->